app/models/playbook.rb

```diff
@@ -10,11 +12,14 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #
-# The class or playbook that the hunter has
-# provides unique abilities to the Hunter
 class Playbook < ApplicationRecord
   has_many :improvements
-  has_many :gears
+  has_many :playbook_gears, dependent: :destroy
+  has_many :gears, through: :playbook_gears
   has_many :moves
   has_many :ratings, dependent: :destroy
+
+  validates :name, presence: true
+  validates :description, presence: true
+  acts_as_taggable_on :categories
```

Comment:
Added a category tag to playbooks here so that the playbooks knew which categories to seed with.